docs/using_huggingface.md

```
@@ -1,118 +1,85 @@
+    downloads. Preview every publication locally with `--dry-run`, then verify
+    the real commit and root alias in the target repository.
 
-```bash
-pip install lanfactory[hf]
-```
+## Install and authenticate
 
-Or install all optional dependencies:
+Install the optional Hub dependency:
 
 ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)